### DISCUSSION TITLE
Does Sqlg work with Java 17?

### DISCUSSION THREAD

**Discussion by pg42819-chris** [pietermartin/sqlg#475] [General]
Will sqlg compile (as a library dependency, not as source) and run with java 17?
If not, is this on the roadmap?

**Comment by pietermartin** [comment]
Yes Sqlg is developed in java 17.
TinkerPop has some concerns regarding java 17, but non of them concern me as I don't make use of gremlin-server and its complexities. https://issues.apache.org/jira/browse/TINKERPOP-2703
